### PR TITLE
Minor edits to Sauce REST API docs

### PR DESCRIPTION
**Pull request by kimsauce**
### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation fix (typos, incorrect content, missing content, etc.)

### Checklist

- [ ] I have read the [contributing](https://github.com/saucelabs/sauce-docs/blob/main/docs/contributing.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/dev/api.md [hunk 1]
```diff
@@ -24,45 +24,47 @@ You can check the current accessibility of any Sauce Labs system on the [Sauce L
...
-
-The request examples throughout the API documentation utilize variables for the authentication credentials. Consider setting your credentials as environment variables using the following commands so you can simply copy the request samples and run them without having to manually pass your credentials for each call.
-
-```jsx "Set Credential Environment Variables"
-$ export SAUCE_USERNAME=<your.user.name>
-$ export SAUCE_ACCESS_KEY=<your access key>
+```bash title="Header Authenticated Request Example"
+curl -L -X GET 'https://api.us-west-1.saucelabs.com/team-management/v1/users/' \
+-H 'Authorization: Basic' \
+-u $SAUCE_USERNAME \
```
Comment (kimsauce) [inline docs/dev/api.md:58]: @sweeneyskirt-sl I wanted to check on the casing for these flags, since normally they're lowercase

Comment (sweeneyskirt-sl) [reply docs/dev/api.md:58]: Hi @kimsauce -- Thanks for the heads up. I will look into it. I feel like references to the credential variables are always all caps, no? See https://docs.saucelabs.com/basics/environment-variables/ for reference.

**Review by sweeneyskirt-sl — APPROVED** [review]
These changes look fine to me.

**Comment by kimsauce** [comment]
Hi @sweeneyskirt-sl , thanks for reviewing! To answer your question (_I feel like references to the credential variables are always all caps, no?_ ), I was referring to the flags (i.e., **-u**, **-k**), not the credential variables.

**Pull request merged**